src/Components/NavBar.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import logo from '../Images/planet.png';
+
+const NavBar = () => {
+  // This styling will be applied to a <NavLink> when the
+  // route that it links to is currently selected.
```

Comment:
Remove the comments to keep the codebase clean

Reply:
Thank you, Will do that